go/libraries/doltcore/dbfactory/gs.go

```diff
@@ -38,7 +40,8 @@ func (fact GSFactory) CreateDB(ctx context.Context, nbf *types.NomsBinFormat, ur
 		return nil, err
 	}
 
-	gcsStore, err := nbs.NewGCSStore(ctx, nbf.VersionString(), urlObj.Host, urlObj.Path, gcs, defaultMemTableSize)
+	bs := blobstore.NewGCSBlobstore(gcs, urlObj.Host, urlObj.Path)
```

Comment:
Here and elsewhere, I find embedded acronyms to be more readable if you only capitalize the first letter. So NewGcsBlobstore

Reply:
consistent case is the preferred go style: https://github.com/golang/go/wiki/CodeReviewComments

```Initialisms
Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

This rule also applies to "ID" when it is short for "identifier" (which is pretty much all cases when it's not the "id" as in "ego", "superego"), so write "appID" instead of "appId".

Code generated by the protocol buffer compiler is exempt from this rule. Human-written code is held to a higher standard than machine-written code.```